spec/index.bs

```diff
@@ -4201,9 +4203,9 @@ dictionary GPUBindGroupLayoutEntry {
 <dl dfn-type=dict-member dfn-for=GPUBindGroupLayoutEntry>
     : <dfn>binding</dfn>
     ::
-        A unique identifier for a resource binding within a
-        {{GPUBindGroupLayoutEntry}}, a corresponding {{GPUBindGroupEntry}},
-        and the {{GPUShaderModule}}s.
+        A unique identifier for a resource binding within the {{GPUBindGroupLayout}}, corresponding
+        to a {{GPUBindGroupLayoutEntry/binding|GPUBindGroupLayoutEntry.binding}} and a [=@binding=]
```

Comment:
This is GPUBindGroupLayoutEntry. Did you mean GPUBindGroupEntry?
```suggestion
        to a {{GPUBindGroupEntry/binding|GPUBindGroupEntry.binding}} and a [=@binding=]
```

Reply:
Yes, I did. Thanks!